homeassistant/components/speedtestdotnet/sensor.py

```diff
@@ -78,10 +77,7 @@ async def async_setup_entry(
     )
 
 
-# pylint: disable-next=hass-invalid-inheritance # needs fixing
-class SpeedtestSensor(
-    CoordinatorEntity[SpeedTestDataCoordinator], RestoreEntity, SensorEntity
-):
+class SpeedtestSensor(CoordinatorEntity[SpeedTestDataCoordinator], RestoreSensor):
```

Comment:
This should also change the implementation in `async_added_to_hass()` to use `async_get_last_sensor_data()` to retrieve sensor value and possibly unit of measurement

Reply:
I think I fixed this now